src/main/res/values/strings.xml

```diff
@@ -34,7 +34,7 @@
 
   <string name="label_authorization_description">Please provide the server location. The next step will require the credentials.</string>
   <string name="label_authorization_title">Authorization using a Hawkular server is necessary.</string>
-  <string name="label_authorization_support">The supported Hawkular version is 1.0.0 Alpha 3.</string>
+  <string name="label_authorization_support">The supported Hawkular version is 1.0.0 Alpha 9.</string>
```

Comment:
I think you can change to Alpha10, as it depends on stuff that was not present on Alpha 9.